specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01-preview/redis.json

```diff
@@ -1759,6 +1772,34 @@
       ],
       "description": "A single Redis item in List or Get Operation."
     },
+    "RedisInstanceDetails": {
+      "properties": {
+        "sslPort": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Redis instance SSL port."
+        },
+        "nonSslPort": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Redis instance Non-SSL port."
+        },
+        "zone": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Availability zone where instance is located."
```

Comment:
+ ' if the cache uses availability zones'

Reply:
Updated description